cmake/O2Dependencies.cmake

```diff
@@ -639,7 +640,8 @@ o2_define_bucket(
     generators_bucket
 
     DEPENDENCIES
-    Base SimulationDataFormat pythia6 pythia8 MathCore
+    #   Base SimulationDataFormat pythia6 pythia8 MathCore RIO Tree
```

Comment:
is this the idea to remove pythia generators completely? Then the `Generators/CMakelists.txt` has to be revised.